src/helics/common/TomlProcessingFunctions.cpp

```diff
@@ -21,6 +21,18 @@ bool hasTomlExtension(std::string_view tomlString)
     return ((ext == "toml") || (ext == "TOML") || (ext == ".ini") || (ext == ".INI"));
 }
 
+bool looksLikeToml(std::string_view tomlString)
+{
+    if (tomlString.empty()) {
+        return false;
+    }
+    if (tomlString.find("[[") != std::string_view::npos) {
+        return true;
+    }
+
+    return (tomlString.find_first_of('=') != std::string_view::npos);
```

Comment:
Looking for an `=` is probably a pretty good indicator of a file being toml; might not even be worth looking for a nested table before hand. Technically I think an empty string is valid toml (but invalid json) -- however the results of parsing it would be pretty boring.

Reply:
The difficulty I ran into is that as part of the json or toml there could be strings with command line arguments containing '='